docs: migrate apps using loopback-component-passport (#5171)
strongloop/loopback-next: examples/passport-login/README.md

- [Prerequisites](#prerequisites)
- [Install](#install-the-example-locally)
- [Tutorial - Facebook](#Try-it-out-with-FaceBook)
- [Tutorial - Google](#Try-it-out-with-Google)

## Overview

This example demonstrates how to use the LoopBack 4 features (like
`@authenticate` decorator, strategy providers, etc) with

> [passport strategies](http://passportjs.org). It includes the oauth2 strategies

Reviewer: ```suggestion
[passport strategies](http://passportjs.org). It includes the OAuth2 strategies
```

Author: agreed

> to interact with external OAuth providers like facebook, google,etc as well as

Reviewer: ```suggestion
to interact with external OAuth providers like Facebook, Google, as well as
```

Author: agreed

- [Prerequisites](#prerequisites)
- [Install](#install-the-example-locally)
- [Tutorial - Facebook](#Try-it-out-with-FaceBook)
- [Tutorial - Google](#Try-it-out-with-Google)

## Overview

This example demonstrates how to use the LoopBack 4 features (like
`@authenticate` decorator, strategy providers, etc) with
[passport strategies](http://passportjs.org). It includes the OAuth2 strategies

> to interact with external OAuth providers like Facebook, Google,etc as well as

Reviewer: ```suggestion
to interact with external OAuth providers like Facebook, Google, etc as well as
```

Author: agreed

local and basic strategies.

You can use this example to see how to,

- Log in or Sign up into a LoopBack App using passport strategy modules
- Log in via external apps like Facebook or link those external profiles with a
  LoopBack user (for example, a LoopBack user can have associated
  Facebook/Google accounts to retrieve pictures).
- Use basic or local passport strategy modules

## Prerequisites

Before starting this tutorial, make sure you have Client-ids/Secrets from third
party apps

- [Facebook](https://developers.facebook.com/apps)
- [Google](https://console.developers.google.com/project)

> - [twitter](https://apps.twitter.com/) **Not yet implemented**

Reviewer: I'm wondering .... if we don't have this yet, maybe it's better to remove this line?

Author: facebook and google oauth is already implemented

1. Click on `Login` from the header menu, You will see various buttons under
   `Other login options`.
   - [Facebook](#Try-it-out-with-FaceBook)
   - [Google](#Try-it-out-with-Google)

>    - `Twitter` - not yet implemented

Reviewer: same here. if we don't have an example yet, maybe we don't need it for now. similarly for other occurrences.

Author: well in the code we have given some url links which are not implemented, so that users can catch on later. It might confuse them, if we dont specify in the docs.

      "failureFlash": true
  }
  ```

### Log in with Google

- Open your browser to the example app with, `http://localhost:3000`

> - Click on 'Log In' from the example app header menu

Reviewer: I believe we usually use back-tick instead. 

```suggestion
- Click `Log In` from the example app header menu
```

Author: agreed
